doc/engineering-reference/src/simulation-models-encyclopedic-reference-003/indoor-swimming-pool.tex

```diff
@@ -152,38 +154,38 @@ \subsection{Conduction through the bottom of the pool}\label{conduction-through-
 \subsection{Makeup pool water supply}\label{makeup-pool-water-supply}
 
 \begin{equation}
-Q_{fw} = m_{fw} \cdot cw \cdot (T_p – T_{fw})
+Q_{fw} = \dot{m}_{fw} \cdot cw \cdot (T_p - T_{fw})
 \end{equation}
 
 where
 
-\(m_{fw}\) = Mass flow rate
+\(m_{fw}\) = Mass flow rate of fresh water (lb·s)
```

Comment:
@mjwitte yes, all of those should be lb/s.  I will correct all of them.  As to why there are IP units in this section, that's a good question.  I will need to go back and look at that.  Maybe the units are not needed.  I don't remember if there was something specific about the equations being from the literature and being developed in IP or if that was just a function of the original NFP having those units.  My preference would be to get rid of the units or have them be in SI.